StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerAddPaymentMethodViewController.swift

```diff
@@ -31,8 +37,50 @@ class CustomerAddPaymentMethodViewController: UIViewController {
     }
     // MARK: - Writable Properties
     private let configuration: CustomerSheet.Configuration
+    private lazy var usBankAccountFormElement: USBankAccountPaymentMethodElement? = {
+        // We are keeping usBankAccountInfo in memory to preserve state
+        // if the user switches payment method types
+        let paymentMethodElement = makeElement(for: selectedPaymentMethodType)
+        if let usBankAccountPaymentMethodElement = paymentMethodElement as? USBankAccountPaymentMethodElement {
+            usBankAccountPaymentMethodElement.presentingViewControllerDelegate = self
+        } else {
+            assertionFailure("Wrong type for usBankAccountFormElement")
+        }
+        return paymentMethodElement as? USBankAccountPaymentMethodElement
+    }()
+
+    var overrideActionButtonBehavior: OverrideableBuyButtonBehavior? {
```

Comment:
nit: Why is the `.LinkUSBankAccount` state capitalized?

I also don't understand why there's only one state for this enum. Not sure if I totally understand the flow, are we overriding the custom "Confirm" button to just say "Continue" and open the bank account linking controller? Do we expect future other states for this button?

Reply:
Why is it capitalized? Copy and pasted from AddPaymentMethodViewController.

Why is there only one state.  From what I remember, because when we implemented USBankAccount for paymentSheet, we thought there would be other use cases for overriding behavior.  Here we are ~1.5 years later and there's still only one use case for overriding the button. I prefer consistency at this point, and refactoring both to remove the enum seems more trouble than it's worth.